lib/vyasa_web/live/media_live/media_bridge.html.heex

```diff
@@ -25,6 +25,13 @@
 <div id="media-player-container" phx-hook="MediaBridge" class="sticky top-0">
   <!-- player -->
   <div id="media-player" class="w-full" role="region" aria-label="Player">
+    <.debug_dump
+      label="Loaded tracklist"
+      tracklist_id={@tracklist_id}
+      is_queue_visible={@is_queue_visible}
```

Comment:
this is the debugdump signal i was talking about -- the toggling for this bool flag isn't entirely right